grafana-init/grafana.py

```diff
@@ -28,16 +28,14 @@
 logger = logging.getLogger(__name__)
 
 GRAFANA_URL = os.environ.get('GRAFANA_URL', 'http://grafana:3000')
-GRAFANA_USERNAME = os.environ.get('GRAFANA_USERNAME', 'mini-mon')
-GRAFANA_PASSWORD = os.environ.get('GRAFANA_PASSWORD', 'password')
+MINI_MON_USER = {'user': 'mini-mon', 'password': 'password', 'email': ''}
```

Comment:
I think you still could get `GRAFANA_USERNAME` and `GRAFANA_PASSWORD` from environment and fill data here so that old behaviour stay the same and users still could overwrite it with `GRAFANA_USERS`.